lib/std/math.zig

```diff
@@ -1096,3 +1096,9 @@ test "compare between signed and unsigned" {
     testing.expect(!compare(@as(u8, 255), .eq, @as(i8, -1)));
     testing.expect(compare(@as(u8, 1), .eq, @as(u8, 1)));
 }
+
+pub const Ternary = enum {
+    yes,
+    no,
+    maybe,
+};
```

Comment:
Counter proposal: use `?bool` instead of this type

Reply:
Fine, I don't have a strong opinion on this.